setup-buildengine.sh

```diff
@@ -592,6 +592,15 @@ if [[ -n $OPT_HACKIT ]]; then
         mersdk@localhost "cat /dev/null | sudo tee /etc/zypp/systemCheck.d/jolla-core.check"
 fi
 
+# df.cache must generated and stored before executing hack-snapshots-cow.sh
+ssh -o UserKnownHostsFile=/dev/null \
+    -o StrictHostKeyChecking=no \
+    -p $SSH_PORT \
+    -i $SSHCONFIG_PATH/vmshare/ssh/private_keys/engine/mersdk \
+    mersdk@localhost "sudo systemctl stop sdk-freespace"
+mkdir vmshare
+cp $SSHCONFIG_PATH/vmshare/df.cache vmshare/
+
```

Comment:
Looking at it a bit closer, I think the systemctl call would be better done from hack-snapshots-cow.sh (it's part of the hack) and vmshare would be better prepared in packVM.